examples/fpga/artya7/README.md

```diff
@@ -64,15 +64,26 @@ This will create a directory `build` which contains the output files, including
 the bitstream.
 
 
-Initial memory parameter SRAMInitFile can be given to FuseSoc to specify which .vmem file to load the design with.
-Example use case includes loading coremark.vmem which can be used for performance/power analysis.
+Initial memory parameter `SRAMInitFile` can be given to FuseSoc to specify which .vmem file to load the design with.
+Example use case includes loading `coremark.vmem` which can be used for performance/power analysis.
 
 Please see [CoreMark README](https://github.com/lowRISC/ibex/blob/master/examples/sw/benchmarks/README.md) for compiling CoreMark.
 
 ```
 fusesoc --cores-root=. run --target=synth --setup --build lowrisc:ibex:top_artya7 --part xc7a100tcsg324-1 --SRAMInitFile=examples/sw/benchmarks/coremark/coremark.vmem
 ```
 
+#### Power Analysis Using Vivado
+
+Setting `FPGAPowerAnalysis` parameter to 1 allows user to run a power analysis using Vivado.
+It uses a post-implementation functional simulation on Vivado to log switching activity.
+Switching activity then get used to generate a detailed power report.
```

Comment:
Grammar nit: This should be "then get**s**" (because "activity" is singular), but I'd probably write "This switching activity is then used...", which is a little more idiomatic and ties the sentence to the previous one.